autoload/crystalline.vim

```diff
@@ -396,6 +396,13 @@ function! crystalline#generate_theme(theme) abort
   let g:crystalline_tab_type_fake_separators = []
 
   for [l:group, l:attr] in items(a:theme)
+    if hlexists('Crystalline' . l:group)
```

Comment:
Hmmm... same problem as in the last MR with `hlexists` here. if `'Crystalline' . l:group` has ever been defined, it won't be added here. I think that means that you can't change the theme at all if you check using `hlexists`.

Reply:
See https://github.com/rbong/vim-crystalline/pull/22#issuecomment-603820010